soap/src/java/com/zimbra/soap/type/WantRecipsSetting.java

```diff
@@ -44,6 +45,8 @@ public static WantRecipsSetting usefulValue(WantRecipsSetting setting) {
             return PUT_SENDERS;
         } else if (WantRecipsSetting.LEGACY_PUT_RECIPS.equals(setting)) {
             return PUT_RECIPIENTS;
+        } else if (WantRecipsSetting.LEGACY_PUT_RECIPS.equals(setting)) {
```

Comment:
please let me know the difference between the conditions at line 46 and 48.

Reply:
Pardon me, I missed updating this condition with _PUT_ALL_.